packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -119,7 +119,9 @@ export class Accordion extends GOVUKFrontendComponent {
     super()
 
     if (!($module instanceof HTMLElement)) {
-      return this
+      throw new TypeError(
+        'Expected `$module` to be an instance of `HTMLElement`'
```

Comment:
Shall we add component name prefixes everywhere and double quotes from https://github.com/alphagov/govuk-frontend/pull/4177?

```suggestion
        `Accordion: $module is not an instance of "HTMLElement"'
```

Or even better, tell the user what the incorrect instance is (e.g. `HTMLTextElement`)

```suggestion
        'Expected `$module` "${$module && 'constructor' in $module ? $module.constructor.name : $module}" is not an instance of `HTMLElement`'
```